R/auditCommonFunctionsBayesian.R

```diff
@@ -19,1941 +19,906 @@
+					}
+				} else if(options[["shadePrior"]] == "shadePriorHypotheses"){
+					if(likelihood == "binomial")
+						plot <- plot + ggplot2::stat_function(fun = dbeta, args = list(shape1 = alphaParameterPrior, shape2 = betaParameterPrior), xlim = c(0, materiality), geom = "area", fill = shade[1]) +
+										ggplot2::stat_function(fun = dbeta, args = list(shape1 = alphaParameterPrior, shape2 = betaParameterPrior), xlim = c(materiality, 1), geom = "area", fill = shade[2])
+					if(likelihood == "poisson")
+						plot <- plot + ggplot2::stat_function(fun = dgamma, args = list(shape = alphaParameterPrior, rate = betaParameterPrior), xlim = c(0, materiality), geom = "area", fill = shade[1]) +
+										ggplot2::stat_function(fun = dgamma, args = list(shape = alphaParameterPrior, rate = betaParameterPrior), xlim = c(materiality, 1), geom = "area", fill = shade[2])
+					if(likelihood == "hypergeometric"){
+						nseq 	<- (N * materiality + 1)
+						xseq1 	<- xseq[1:nseq]
+						bars1 	<- data.frame(x = xseq1, y = jfa:::.dBetaBinom(xseq1, N = N - n + k, shape1 = alphaParameterPrior, shape2 = betaParameterPrior))
+						plot 	<- plot + ggplot2::geom_bar(data = bars1, stat = "identity", fill = shade[1])
+						xseq2 	<- xseq[(nseq + 1)]:N
+						bars2 	<- data.frame(x = xseq2, y = jfa:::.dBetaBinom(xseq2, N = N - n + k, shape1 = alphaParameterPrior, shape2 = betaParameterPrior))
+						plot 	<- plot + ggplot2::geom_bar(data = bars2, stat = "identity", fill = shade[2])
+					}
+				}
+			} 
+		}
+
+		if(options[["priorPlotAdditionalInfo"]])
+			plot <- plot + ggplot2::geom_point(mapping = ggplot2::aes(x = x, y = y), data = mPrior, size = 3, shape = 21, stroke = 2, color = "black", fill = rgb(0.9, 0, 0, 1)) +
+							ggplot2::geom_point(mapping = ggplot2::aes(x = x, y = y), data = ePrior, size = 3, shape = 21, stroke = 2, color = "black", fill = "grey")
+
+		if(options[["priorPlotAdditionalInfo"]] && options[["priorPlotExpectedPosterior"]])
+			plot <- plot + ggplot2::geom_segment(x = 0, xend = bPost, y = max(dPlot$y) * 1.1, yend = max(dPlot$y) * 1.1, 
+													linetype = 1, size = 1, arrow = ggplot2::arrow(length = ggplot2::unit(0.03, "npc"), angle = 90)) +
+							ggplot2::geom_segment(x = bPost, xend = 0, y = max(dPlot$y) * 1.1, yend = max(dPlot$y) * 1.1, 
+													linetype = 1, size = 1, arrow = ggplot2::arrow(length = ggplot2::unit(0.03, "npc"), angle = 90))
+
+		jfaTheme <- ggplot2::theme(axis.ticks.y = ggplot2::element_blank(),
+									axis.title.y = ggplot2::element_text(margin = ggplot2::margin(t = 0, r = -5, b = 0, l = 0)),
+									legend.key.size = ggplot2::unit(2.5, "line"),
+									legend.position = "top")
+	
+		plot <- jaspGraphs::themeJasp(plot) + jfaTheme
+		figure$plotObject <- plot
+  	}
+
+	if(options[["explanatoryText"]]){
+		distribution 	<- base::switch(parentOptions[["likelihood"]], "poisson" = "gamma", "binomial" = "beta", "hypergeometric" = "beta-binomial")
```

Comment:
gettext?

Reply:
Also, this exact `switch` statement seems familiar to me, maybe put it in a function?